### DISCUSSION TITLE
Install failure

### DISCUSSION THREAD

**Discussion by chellea** [dorssel/usbipd-win#487] [General]
When I run...

sudo apt install linux-tools-virtual hwdata
sudo update-alternatives --install /usr/local/bin/usbip usbip `ls /usr/lib/linux-tools/*/usbip | tail -n1` 20

I get the following...

Building dependency tree
Reading state information... Done
The following package was automatically installed and is no longer required:
  libfwupdplugin1
Use 'sudo apt autoremove' to remove it.
The following additional packages will be installed:
  libdw1 linux-tools-5.4.0-107 linux-tools-5.4.0-107-generic linux-tools-common
The following NEW packages will be installed:
  hwdata libdw1 linux-tools-5.4.0-107 linux-tools-5.4.0-107-generic linux-tools-common linux-tools-virtual
0 upgraded, 6 newly installed, 0 to remove and 0 not upgraded.
Need to get 6047 kB of archives.
After this operation, 27.8 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
**Abort.**

Any thoughts?

Thank you.

**Comment by dorssel** [comment]
Maybe you need a `sudo apt update` first? Might be that your package list is outdated... (on the other hand, I would have expected a more informative error)
